pkg/reconciler/v1alpha1/clusteringress/config/store_test.go

```diff
@@ -32,12 +32,10 @@ func TestStoreLoadWithContext(t *testing.T) {
 	store.OnConfigChanged(istioConfig)
 	config := FromContext(store.ToContext(context.Background()))
 
-	t.Run("load istio", func(t *testing.T) {
```

Comment:
Why'd you remove the `t.Run`?

Reply:
the test name isn't changing inside a for loop, I didn't see any purpose for the t.Run